src/scicookie/{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -46,9 +46,9 @@ repos:
           - python
 {% endif %}
 {%- if cookiecutter.use_ruff_linter == 'yes' %}
-      - id: ruff-linter
-        name: ruff-linter
-        entry: ruff check --fix
+      - id: ruff check
```

Comment:
@YurelyCamacho , could you move back to the original version of this file please?
this was already fixed on the main branch.
thanks!

Reply:
@xmnlab Done